docs/TUTORIAL.md

```diff
@@ -414,8 +408,8 @@ Enter a password for the encrypted RSA key (/path/to/snapshot_key):
 Enter a password for the encrypted RSA key (/path/to/timestamp_key):
 >>> repository.timestamp.load_signing_key(private_timestamp_key)
 
->>> repository.dirty_roles()
-Dirty roles: ['root', 'snapshot', 'targets', 'timestamp']
+# Mark roles for metadata update (see #XXX, #958)
+>>> repository.mark_dirty(['snapshot', 'targets', 'timestamp'])
```

Comment:
Should root be marked dirty here also? It was in the output of `dirty_roles()` above.

Reply:
No. The output was wrong before in TUTORIAL.md (test_tutorial.py had it right). At this point in the tutorial, root does not need to be re-written.

Reply:
But please double check! :)